core/core.md

```diff
@@ -130,3 +130,15 @@ As noted in Non-Goal #3 in the [goals doc](../dogma/goals.md), this is not somet
 As mentioned above, for an honest members to successfully connect to the rest of the honest community; they must not be eclipsed by dishonest Hosts. If an honest member has no choice but to access the community through a dishonest Host, then Fora can make no guarantee that the member can be protected from banned members and banned content.
 
 Thus the Fora protocol assumes that the peer-to-peer network is sufficiently distributed and decentralized such that no member can be fully eclipsed by dishonest Hosts.
+
+## Dishonest Members
+
+Members who consistently or flagrantly break community rules can be banned by Enforcers. However, these bad actors can simply create a new identity and try to enter the community again. Thus, Fora communities facing these Sybil attacks, should tighten their admission requirements to prevent banned users from re-entering the community with freshly minted identities.
+
+This can be achieved either by evaluating whether the proposed member is an established member in other Fora communites, or by relying on attestations from members who are already in the community.
```

Comment:
Someone can also create a Fora community where the did must map back to a real world identity, through some mechanism. Attestations are one. Or the admittors could enforce KYC with some form of ID if the country/community.

Dependent on the context of the community. Maybe helpful to mention the flexibility here?

Reply:
yes for example, a github account might be sufficient KYC

Reply:
Addressed